Makefile

```
@@ -16,7 +16,7 @@ clean:
 test: test-local
 	set -eu; \
 	for p in $(PACKAGES); do \
-		if $p = user && go version | grep -qv go1.18; then \
+		if $p = userns && go version | grep -qv go1.18; then \
```

Wait, this syntax is funky -- `if $p = userns` is going to run `$p` as an executable with arguments `= userns` and since `$p` is a directory, will probably _always_ be false and thus this block never runs?

---

```suggestion
		if [ "$p" = userns ] && go version | grep -qv go1.18; then \
```

(probably meant something like this here and in the other loops in this file)

---

Also, we _only_ run the tests on go1.18?  That seems strange? (I'm likely missing some context)